lib/matplotlib/backend_bases.py

```diff
@@ -2791,6 +2785,12 @@ def mouse_move(self, event):
             except (ValueError, OverflowError):
                 pass
             else:
+                artists = event.inaxes.hitlist(event)
+                if event.inaxes.patch in artists:
+                    artists.remove(event.inaxes.patch)
+                artists.sort(key=lambda x: x.zorder)
+                if artists:
+                    s += artists[-1].get_zdata(event)
```

Comment:
@pelson  - By default, there isn't enough room in the cursor window for more than one.

If nothing else, Tkinter does odd things when the string displayed in the cursor becomes larger than the box it's in.  (The window resizes slightly, and it gives a visually jarring "jump" effect.)